composer.json

```diff
@@ -19,19 +18,21 @@
     ],
     "require": {
         "php": ">=5.5.9",
+        "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/orm": "^2.4.8",
-        "symfony/framework-bundle": "~2.8|~3.0|^4.0",
-        "symfony/form": "~2.8|~3.0|^4.0"
+        "symfony/form": "^3.4|^4.0|^5.0",
+        "symfony/framework-bundle": "^3.4|^4.0|^5.0"
     },
     "require-dev": {
+        "doctrine/mongodb-odm": "^1.1|^2.0",
         "doctrine/mongodb-odm-bundle": "^3.0",
```

Comment:
Try to allow mongodb-odm-bundle 4 too

Reply:
I tried, but mongo-odm-bundle 4 removed https://github.com/doctrine/DoctrineMongoDBBundle/blob/3.5/DataCollector/PrettyDataCollector.php in 4.*

And brokes our tests, maybe I could change the tests :roll_eyes:

Reply:
But in 3.* we have PrettyDataCollector.php and StandardDataCollector.php
In 4.* we have only CommandDataCollector.php

Reply:
Well, I can't find anything in mongo-odm-bundle mentioting the removal of PrettyDataCollector. Anyway, can't you just use CommandDataCollector instead?

Reply:
I opened an issue to see if we can get some feedback from bundle maintainers: https://github.com/doctrine/DoctrineMongoDBBundle/issues/606

Reply:
I tried to use CommandDataCollector, but ODM bundle brokes a lot of codes :/ 

(Who use ODM Bundle really ? :D )

Reply:
There's a hard BC break regarding data collectors from 3.x to 4.x. Unfortunately, we can't really fix that as 3.x does not know about the commands. This is a side effect of the driver change underneath the hood.

As for the original issue, there's no point adding `^2.0` for ODM if you don't allow 4.0 for the bundle: 3.x is not compatible with ODM 2.0 and will prevent installing it anyways.

Last but not least, you should also change the constraint for ODM to `^1.3|^2.0`, as 1.1 has been EOL'd a while ago and doesn't contain the forward compatibility and deprecation layer for 2.0.